wifimgr.py

```diff
@@ -119,62 +222,58 @@ def handle_root(client):
     wlan_sta.active(True)
     ssids = sorted(ssid.decode('utf-8') for ssid, *_ in wlan_sta.scan())
     send_header(client)
+    client.sendall("<html>")
+    client.sendall(html_head)
     client.sendall("""\
-        <html>
-            <h1 style="color: #5e9ca0; text-align: center;">
-                <span style="color: #ff0000;">
-                    Wi-Fi Client Setup
-                </span>
-            </h1>
-            <form action="configure" method="post">
-                <table style="margin-left: auto; margin-right: auto;">
-                    <tbody>
+            <body>
+                <form action="configure" method="post">
+                    <h1>
+                        Wi-Fi Client Setup
+                    </h1>
+                    <h3>
+                        SSIDs (click to select)
+                    </h3>
+                    <ul>
     """)
     while len(ssids):
         ssid = ssids.pop(0)
         client.sendall("""\
-                        <tr>
-                            <td colspan="2">
-                                <input type="radio" name="ssid" value="{0}" />{0}
-                            </td>
-                        </tr>
-        """.format(ssid))
+                        <a onclick="document.getElementById('ssid').value = '{ssid}';" href="javascript:void(0)">
+                            <li>
+                                {ssid}
+                            </li>
+                        </a>
```

Comment:
is this valid html?: `a li text /li /a`

or should it be rather?: `li a text /a /li`

Reply:
A is not `display: block` ( or similar ), so I see no problem.